guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -4,6 +4,36 @@
 :package-manager: dnf
 
 Use this procedure to enable the repositories that are required to install {ProductName}.
+Select the operating system and version you are installing on:
+
+* xref:#repositories-capsule-rhel-9[{RHEL} 9]
+* xref:#repositories-capsule-rhel-8[{RHEL} 8]
+
+[id="repositories-capsule-rhel-9"]
```

Comment:
```suggestion
[id="repositories-{smart-proxy-context}-rhel-9"]
```

Same for line 35. source: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L122